yw_website/website/templates/registration/login.html

```diff
@@ -1,47 +1,10 @@
+<html>
 {% load staticfiles %}
+{% include 'includeable/navbar.html' %}
```

Comment:
it's better to put all html below the stylesheets since html loads content in cascading order.